_includes/image.html

```diff
@@ -18,7 +18,7 @@
             {% if source_link %}
               <a href="{{ source_link }}" target="_blank" rel="noreferrer">
             {% endif %}
-            <img class="img-thumbnail img-fluid"
+            <img class="img-thumbnail img-fluid border-0"
```

Comment:
I'm strongly in favor of keeping the border styling on images. 😊 

I think it looks much better. 😄

Reply:
Brought border back.